Example/ios/Podfile

```diff
@@ -8,13 +8,17 @@ target 'ScreensExample' do
 
   pod 'RNScreens', :path => '../../'
   
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # to enable hermes on iOS, change `false` to `true` and then install pods
+    :hermes_enabled => false
+  )
 
   # Enables Flipper.
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
-  use_flipper!
+  # you should disable the next line.
+  use_flipper!()
```

Comment:
This isn't needed and in the Ruby ecosystem not using parens for no-arg functions is considered more idiomatic.

Reply:
This is most prolly taken from the migration guide: https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.0, where it is included. I don't have a strong opinion if it should be here though.

Reply:
Yup, this is just a nit, feel free to resolve it however you want.

Reply:
I'll revert the last commit with upgrading the example project to react-native to 0.64.0 and create separate PR for this to upgrade the whole library.

Reply:
Resolved by https://github.com/software-mansion/react-native-screens/pull/827/commits/f7bd96e2b94c981c76f948d942cf80149076ef8b 🍉